src/feature.js

```diff
@@ -88,10 +96,16 @@ class Feature {
     browser.pageMonitor.onPageDOMContentLoaded.addListener(async (tabId, data) => {
       const tabInfo = TabRecords.getOrInsertTabInfo(tabId);
       const location = /[^?]*/.exec(data.completeLocation)[0];
```

Comment:
Unrelated to this commit, but it would be nice to have a short comment on what this does :)